src/vulkan/image.cc

```diff
@@ -144,7 +140,7 @@ Result Image::CopyToHost(VkCommandBuffer command) {
                          GetHostAccessibleBuffer(), 1, &copy_region);
 
   MemoryBarrier(command);
-  return {};
+  return;
```

Comment:
nit: return at end of method not needed.

Reply:
done